test/e2e/framework.go

```diff
@@ -386,3 +387,20 @@ func kubectlExec(namespace string, podName, containerName string, args ...string
 	err := cmd.Run()
 	return stdout.Bytes(), stderr.Bytes(), err
 }
+
+// Wrapper to ginkgo functions for e2e tests which can be easily filtered via focus tags.
+func KubeIt(text string, body interface{}, timeout ...float64) bool {
+	tname := "["+KubeE2ETestTag+"] "+text;
+	if len(timeout) == 0 {
```

Comment:
Only one timeout is worth supporting, since the variadic is just a way to "not" send anything. 
But I think supporting a full variadic array is completely unnecessary.